apps/io-func-wallet-solution-backend/src/encoders/wallet-instance-attestation.ts

```diff
@@ -0,0 +1,73 @@
+import * as t from "io-ts";
+import * as E from "io-ts/lib/Encoder";
+import { JwkPublicKey } from "../jwk";
+import { WalletInstanceAttestationPayload } from "../wallet-instance-attestation";
+
+const AlgValueSupported = t.type({
+  alg_values_supported: t.array(t.string),
+});
+
+export const WalletInstanceAttestationJwtModel = t.type({
+  iss: t.string,
+  sub: t.string,
+  type: t.literal("WalletInstanceAttestation"),
+  policy_uri: t.string,
+  tos_uri: t.string,
+  logo_uri: t.string,
+  asc: t.string,
+  cnf: t.type({
+    jwk: JwkPublicKey,
+  }),
+  authorization_endpoint: t.literal("eudiw"),
+  response_types_supported: t.array(t.string),
+  vp_formats_supported: t.type({
+    jwt_vp_json: AlgValueSupported,
+    jwt_vc_json: AlgValueSupported,
+  }),
+
+  request_object_signing_alg_values_supported: t.array(t.string),
+  presentation_definition_uri_supported: t.boolean,
+});
+
+export type WalletInstanceAttestationJwtModel = t.TypeOf<
+  typeof WalletInstanceAttestationJwtModel
+>;
+
+export const WalletInstanceAttestationToJwtModel: E.Encoder<
+  WalletInstanceAttestationJwtModel,
+  WalletInstanceAttestationPayload
+> = {
```

Comment:
**question**: Maybe we can explain what we mean with `WalletInstanceAttestationJwtModel` and what we intend its use for.

Reply:
I add a comment!